### PR TITLE
README.md: Update intro based on content from MeasurementLink manual

### PR DESCRIPTION
**Pull request by bkeryan**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-python/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Update description of MeasurementLink Support for Python based on content from https://www.ni.com/docs/en-US/bundle/measurementlink/page/measurementlink.html

This will be cherry-picked into releases/1.1.

### Why should this Pull Request be merged?

Consistency with MeasurementLink Manual.

### What testing has been done?

None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,10 @@
 
 ## Introduction
 
-MeasurementLink Support for Python (`ni-measurementlink-service`) is a Python framework that enables measurement developers to quickly create Python measurements and run them as a service (gRPC).
+MeasurementLink Support for Python (`ni-measurementlink-service`) is a Python
+framework that helps you create reusable measurement plug-ins. Deploy your
+measurement plug-ins to perform interactive validation in InstrumentStudio and
+automated testing in TestStand.
```
Comment (bkeryan) [inline README.md:27]: @nstokes-nati This is an abridged version of the 1st paragraph from the MeasurementLink Overview page. Is that okay?

**Review by dixonjoel — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -21,7 +21,10 @@
 
 ## Introduction
 
-MeasurementLink Support for Python (`ni-measurementlink-service`) is a Python framework that enables measurement developers to quickly create Python measurements and run them as a service (gRPC).
+MeasurementLink Support for Python (`ni-measurementlink-service`) is a Python
```
Comment (bkeryan) [inline README.md:24]: On second thought, we should still keep a reference to "service" and "gRPC"

Comment (bkeryan) [reply README.md:24]: "helps you create reusable measurement plug-ins using gRPC services"

File: README.md (hunk 1 above)
Comment (bkeryan) [reply README.md]: I'm merging this PR now.

**Pull request merged**